go/general_style_guide.md

```diff
@@ -7,3 +7,25 @@
 It is important to follow this guide. It makes review process easier and
 faster. It also makes our codebase consistent, easier to navigate, and
 understand.
+
+## Code Quality
+
+To prevent shipping problematic or unsafe code, [`architect-orb`](https://github.com/giantswarm/architect-orb)'s [`go-test`](https://github.com/giantswarm/architect-orb/blob/master/src/commands/go-test.yaml) and [`go-test-legacy`](https://github.com/giantswarm/architect-orb/blob/master/src/commands/go-test-legacy.yaml) commands include running several linting tools via [`golangci-lint`](https://github.com/golangci/golangci-lint).
```

Comment:
Can you wrap the lines like in the rest of this page?

Reply:
:heavy_check_mark: